Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -464,10 +473,14 @@
 		6E2307ED1D22083C0031C598 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = "Brand Assets";
 				ENABLE_BITCODE = NO;
-				FRAMEWORK_SEARCH_PATHS = "$(inherited)";
+				FRAMEWORK_SEARCH_PATHS = (
```

Comment:
Was because of the CodeSignatureExtraction framework. But don't hurt to leave it. Also I harmonized to all apps and it's now at the project level ;)